src/__experimental__/components/radio-button/radio-button.component.js

```diff
@@ -106,9 +91,9 @@ RadioButton.propTypes = {
   }
 };
 
-RadioButton.defaultProps = {
-  onChange: () => { },
+const Component = React.memo(RadioButton);
```

Comment:
Again, why not `export default React.memo(RadioButton)`

Reply:
I think this was so snapshots matched in GAC. Can change as long as defaultProps are still applied.

Reply:
changed